src/class-names/index.js

```diff
@@ -76,6 +76,8 @@ export default async function getClassNames(cwd = process.cwd(), { onChange = ()
               })
             )
           )
+        } catch (error) {
+          throw error
```

Comment:
`try { ... } finally { ... }` won't swallow the error.

Reply:
Oh cool sorry, my bad 👍 Never seen that before. Feel free to change it back

Reply:
This doesn't work on yarn workspaces, but works on root project...

Reply:
Sorry, it does work, but make sure to include: 

```
{ 
purge: ['./pages/**/*.{js,jsx,ts,tsx}'],
}
```

in your tailwind.config.js